sklearn/metrics/regression.py

```diff
@@ -235,6 +238,10 @@ def mean_squared_error(y_true, y_pred,
     >>> mean_squared_error(y_true, y_pred, multioutput=[0.3, 0.7])
     ... # doctest: +ELLIPSIS
     0.825...
+    >>> y_true = [[0.5, 1],[-1, 1],[7, -6]]
```

Comment:
put this example before the multioutput examples and try to make use of existing data